feat(ins): add troubleshooting index (#4580)
scaleway/docs-content: pages/instances/troubleshooting/index.mdx

---
meta:
  title: Instances - Troubleshooting
  description: Find troubleshooting resources for resolving issues with Scaleway Instances.
content:

>   paragraph: Find troubleshooting resources for resolving issues with Scaleway Instancdes.

Reviewer: ```suggestion
  paragraph: Find troubleshooting resources for resolving issues with Scaleway Instances.
```

dates:
  posted: 2025-03-10
categories:
  - instances
---

<ProductHeader
  productName="Instances troubleshooting"
  productLogo="instance"

>   description="Troubleshoot the most common issues with Scaleway Instances by following our guides."

Reviewer: ```suggestion
  description="Learn how to fix the most common issues with Scaleway Instances."
```

/>

## Featured Pages

<Grid>
    <DefaultCard
      title="Instances FAQ"

>       description="General info on Instances"

Reviewer: ```suggestion
      description="General info on Instances."
```

      url="/faq/instances/"
      label="See more"
    />
    <DefaultCard
      title="Choosing the best Scaleway Instance type for your workload"

>       description="Different Instance types have different prices and are designed for different use cases."

Reviewer: ```suggestion
      description="Compare pricing and functioning to choose the best Instance type for your use case."
```

      url="/instances/reference-content/choosing-instance-type/"
      label="See more"
    />
    <DefaultCard
      title="Instances datasheet"

>       description="Compare the technical specifications with the Instances datasheet."

Reviewer: ```suggestion
      description="Compare technical specifications with the Instances datasheet."
```

      url="/instances/reference-content/instances-datasheet/"
      label="See more"
    />
</Grid>
<Grid>
    <DefaultCard
      title="Understanding the differences between ARM and x86 Instances"
      description="Understand the architectural differences between x96 and ARM CPUs."
      url="/instances/reference-content/understanding-differences-x86-arm/"
      label="See more"
    />
    <DefaultCard
      title="Understanding Scaleway Instance pricing"
      description="Learn about pricing and understand how you will be billed when using Instances."
      url="/instances/reference-content/understanding-instance-pricing/"
      label="See more"
    />
    <DefaultCard
      title="Instances internet and Block Storage bandwidth overview"

>       description="Learn about the technical characteristics of internet and Block Storage bandwidth of the different Instance types."

Reviewer: ```suggestion
      description="Learn about the technical aspects of internet and Block Storage bandwidth of Instance types."
```
